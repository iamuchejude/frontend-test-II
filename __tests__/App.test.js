import React from 'react';
import { shallow } from 'enzyme';
import App from '../src/App';

describe('App components', () => {
  test('should render app component', () => {
    const wrapper  = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  })
});
