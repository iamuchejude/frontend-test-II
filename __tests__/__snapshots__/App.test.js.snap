// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App components should render app component 1`] = `
<div>
  <AppRoutes />
</div>
`;
